airflow/cli/cli_config.py

```diff
@@ -769,6 +769,9 @@ def string_lower_type(val):
     ("-L", "--access-logformat"),
     help="The access log format for gunicorn logs",
 )
+ARG_FASTAPI_API_APPS = Arg(
+    ("--apps",), help="Applications to run. Default is all", type=string_list_type, default=["ui", "task-sdk"]
```

Comment:
Do we want a "public" option too. Which onylyruns the public rest api?

"ui" should definitely run the public rest api anyway